ceph-dev-trigger/config/definitions/ceph-dev-trigger.yml

```diff
@@ -101,7 +101,7 @@
                     DISTROS=centos8
                     FLAVOR=crimson
       # build master on:
-      # default: focal bionic centos8 leap15
+      # default: focal centos8 leap15
```

Comment:
I don't think master is built on `leap15` either.

Reply:
thanks. i added a commit for it. as it's not quite related to the bionic change.